.github/workflows/npm-publish.yaml

```diff
@@ -0,0 +1,23 @@
+name: npm-publish
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        description: The version you intend to publish (eg x.y.z)
```

Comment:
We have to be very cautious with fat fingers when publishing to npmjs.com. It is basically impossible to unpublish a package without the help of npm support, so an inadvertent `npm-publish` with `v11.3.2` would be pretty difficult to fix up.

But if I read the code correctly, we should fail if the version isn't already released, is that correct?

Reply:
It will fail if a proper version isn't provided. I believe there is a little room for this to fail though, so I'll add a few extra checks.